### PR TITLE
Move solutions file to master - #5

### LINKED ISSUE #5: move solutions into a folder in `master` branch
As requested in the other repos, [please don't make solutions branches](https://github.com/foundersandcoders/master-reference/issues/456)
+ https://github.com/foundersandcoders/pg-walkthrough/pull/16#issuecomment-318824762
+ https://github.com/foundersandcoders/pg-workshop/issues/26

This should go in a folder in the `master` branch

### PR DESCRIPTION
**Pull request by macintoshhelper**
Fixes #5

### CODE REVIEW

**Review by jsms90 — APPROVED** [review]

**Comment by jsms90** [comment]
@ConchobarMacNessa @pbywater Should have someone who has actually user tested this workshop be the person to merge the solutions in

**Review by jsms90 — CHANGES_REQUESTED** [review]
The readme has been removed?

File: README.md [hunk 1]
```diff
@@ -1 +1,168 @@
...
+6. Imagine an API request is made for blog posts with the IDs `3 and 6`. Using `WHERE` and [`IN`](https://www.w3schools.com/sql/sql_in.asp), show the blog posts' user id and text content.
+
+**Expected Result**
+
+user\_id | text\_content
+--- | ---
+3 | Far stairs now coming bed oppose hunted become his. You zealously departure had procuring suspicion. Books whose front would purse if be do decay.
+6 | Etiam in est nec neque dapibus pretium in in lectus. Proin consequat velit quis magna aliquam tristique. Sed ultricies nulla vel feugiat mattis. Aliquam erat volutpat. Aliquam ac vehicula diam, eget ultricies nisi.
+
+7. Using [`CASE WHEN`](https://www.postgresql.org/docs/7.4/static/functions-conditional.html), get a list of user ids from the `users` table and label a new row, `teenager` with `true` or `false`.
```
Comment (shiryz) [inline README.md]: Adding a hint for AS would help, they haven't seen how to create a new column before

File: README.md [hunk 2]
```diff
@@ -1 +1,168 @@
...
+
+**Expected Result**
+
+user\_id |
+--- |
+5 |
+
+**Bonus Question**
+
+10. Using [`INSERT INTO`](https://www.w3schools.com/sql/sql_insert.asp), add a comment with the text `Interesting post` from the user\_id of `3` to the blog post containing the text `Peculiar` (reply\_to will be null), then run `SELECT text_content FROM post_comments WHERE id = 2;` to test for the expected result.
```
Comment (shiryz) [inline README.md]: Add a hint for sub-queries, as in *hint*: search sub-queries

**Review by shiryz — CHANGES_REQUESTED** [review]
Great work @macintoshhelper , did not catch these changes yesterday, it shouldn't take too long

**Review by jsms90 — APPROVED** [review]

**Review by jsms90 — CHANGES_REQUESTED** [review]
@macintoshhelper It currrently looks like a whole readme file is being added, which it's not, because the readme file is already in the master branch.

You need to merge this branch with master.

**Review by shiryz — APPROVED** [review]

**Comment by jsms90** [comment]
@macintoshhelper The `init.sql` file is also being removed with this PR

**Comment by m4v15** [comment]
closing as superseded by #18

**Pull request closed without merging**